lib/tasks/after_tasks.rake

```diff
@@ -517,5 +517,26 @@ namespace :After do
 
     puts "Finished reindexing tags on hidden and unrevealed works"
   end
+
+  desc "Convert user kudos from users with the official role to guest kudos"
+  task(convert_official_kudos: :environment) do
+    official_role = Role.find_by(name: "official")
+    official_users = RolesUser.where(role_id: official_role.id).map(&:user) if official_role.present?
+    if official_role.blank? || official_users.blank?
+      puts "No official users found"
+    else
+      official_users.each do |user|
+        kudos = user.kudos
+        next if kudos.blank?
+
+        puts "Updating #{kudos.size} kudos from #{user.login}"
+        user.kudos.each do |kudo|
```

Comment:
You can use `user.remove_user_from_kudos`.

Reply:
...I might have forgotten #4264 was done. (Why did I think that was the issue we were just talking about whether we could use `touch` with?)

Reply:
We _were_ heavily referencing the discussions on that PR.

Reply:
Oh, yay, I'm not _totally_ losing it 🎉